specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json

```diff
@@ -193,6 +193,64 @@
           }
         }
       }
+    },
+    "/restore/key/{keyName}": {
```

Comment:
The traditional Key Vault key restore api is rooted at /keys/restore. Is it intended to deviate from the /keys prefix for selective restore? We could also do /keys/selectiverestore/{keyName}

Reply:
I'm thinking of changing this to /keys/{keyName}/restore: and moving this to keys.json.
so this will not interfere the existing keyvault's key restore from a single key (ie., /keys/restore) where a whole key is passed to restore.